src/npm/prepend-core.js

```diff
@@ -1,11 +1,6 @@
-module.exports = function(opalParam, xmlHttpRequestParam) {
-  var Opal = opalParam || require('opal-runtime').Opal;
-
-  if (xmlHttpRequestParam !== false) {
-    var XMLHttpRequest = xmlHttpRequestParam || require('xmlhttprequest').XMLHttpRequest;
-    // Define overrideMimeType, not define by default in wrapper
-    XMLHttpRequest.prototype.overrideMimeType = function() {};
```

Comment:
I think this is no longer needed because `XMLHttpRequest` is only used when running on browser.
We might need to improve the detection of `XMLHttpRequest` because according to @anthonny, `XMLHttpRequest` is only available on the `window` object (when using module loader like Browserify or Webpack): https://github.com/asciidoctor/asciidoctor.js/issues/178#issuecomment-205062716